### ISSUE TITLE
[bug] web-api-no-leaked-event-listener doesn't recognize usage when React is imported as a namespace

### ISSUE THREAD

**Issue by Paatus** [Rel1cx/eslint-react#1228]
### Describe the bug

When using react with a namespace import (`import * as React from 'react'`), `web-api-no-leaked-event-listener` does not report about missing removeEventListeners

### Reproduction
```
import * as React from 'react';

function useCustomHook() {
  React.useEffect(() => {
    const handleClick = () => {
      console.log("clicked");
    };
    document.addEventListener("click", handleClick);
    //                                 ^^^^^^^^^^^^
    //                                 - No error is reported here
  }, []);
}
```

### Expected behavior
An error should be reported 

### Platform and versions

```Text
@eslint-react/eslint-plugin: 1.53.0
react: 19.1.0
react-dom: 19.1.0
eslint: 9.27.0
```

### Stack trace

```Text

```

### Additional context

It works as expected if the useEffect is imported like this `import { useEffect } from 'react'`

**Comment by Paatus** [comment]
I'd be happy to take a look at this

**Comment by Rel1cx** [comment]
> I'd be happy to take a look at this

Hi @Paatus, many thanks for offering to help! I just pushed a fix for this. I really appreciate your willingness to contribute!

**Comment by Paatus** [comment]
> > I'd be happy to take a look at this
> 
> Hi [@Paatus](https://github.com/Paatus), many thanks for offering to help! I just pushed a fix for this. I really appreciate your willingness to contribute!

Awesome, thank you!
